src/server.c

```diff
@@ -3650,16 +3663,7 @@ int processCommand(client *c) {
     /* If we're executing a script, try to extract a set of command flags from
      * it, in case it declared them. Note this is just an attempt, we don't yet
      * know the script command is well formed.*/
```

Comment:
i think the comment should move to the new function together with the code

Reply:
ok, moved